### PR TITLE
Updated almost all section text to be more consistent and readable

### PR DESCRIPTION
**Pull request by LowKeyNerd**
Removed confusing jargon, double speak, added grammatical fixes and consistency, cleaned up some minor formatting, etc.

### CODE REVIEW

File: app/_sections/Architecture.tsx [hunk 1]
```diff
@@ -33,35 +33,36 @@ export default function Architecture() {
...
             </DescriptionBox>
             <DescriptionBox title="Watcher" img="rosen-architecture-watchers.png" right>
-                <Text>A watcher is an entity tasked with monitoring a blockchain and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. It’s open for anyone to join.</Text>
+                <Text>Watchers are entities tasked with monitoring a given blockchain (e.g., Cardano, Bitcoin, Ethereum, etc.) and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of Watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. Anyone can participate as a Watcher once
+                     allocating enough RSN and native currency of the bridged chain (e.g., 1000 ERG and 1000 RSN).</Text>
```
Comment (fatemeh-ra) [inline app/_sections/Architecture.tsx:49]: This sentence is incorrect. All watchers should put collateral in the form of ERG and RSN. Besides the collateral amount is not fixed yet.  
The collateral is explained in later sections and is not required here. This text part became a little long.

File: app/_sections/Features.tsx [hunk 2]
```diff
@@ -5,37 +5,41 @@ export default function Features() {
...
             title: "Multi-Layer Protection",
-            description: "Safeguarded by two distinct layers of security",
+            description: "Safeguarded by two distinct layers of security.",
             icon: <i className="uil uil-layer-group"/>,
         },
         {
-            title: "Ergo-Centric",
-            description: "Uses Ergo as a secure and cost efficient hub, reducing the need for several smart contracts on every chain. Facilitates linear growth complexity.",
+            title: "Smart Contract Reduction",
+            description: "Uses Ergo as a secure and cost efficient hub, reducing the need for several smart contracts on every bridged chain. Facilitates linear growth complexity.",
```
Comment (fatemeh-ra) [inline app/_sections/Features.tsx:13]: It is already long. Can you summarize it a little?

File: package.json [hunk 3]
```diff
@@ -11,15 +11,15 @@
...
+    "@types/react-dom": "^18.2.6",
     "autoprefixer": "10.4.14",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.9",
     "next": "13.4.9",
     "postcss": "8.4.25",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```
Comment (fatemeh-ra) [inline package.json:22]: Don't change these

Comment (LowKeyNerd) [reply package.json:22]: This was changed automatically - you are welcome to change it back. It just allows for folks like me to contribute easier as it opens up version to this versioning or higher.

File: app/_sections/Architecture.tsx (hunk 1 above)
Comment (LowKeyNerd) [reply app/_sections/Architecture.tsx:49]: I am not going to pretend to understand the intricacies of Rosen - I took what was there and tried to make it make more sense to the casual read (much like myself). If I was unable to do so, I think that it's pretty indicative there needs to be more context.

Does this make more sense? 

Watchers are entities tasked with monitoring a given blockchain (e.g., Cardano, Bitcoin, Ethereum, etc.) and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of Watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. Anyone can participate as a Watcher once allocating enough RSN and and ERG).</Text>

File: app/_sections/Features.tsx (hunk 2 above)
Comment (LowKeyNerd) [reply app/_sections/Features.tsx:13]: Try: Leverages Ergo for security and cost savings, minimizing smart contracts on bridged chains and ensuring scalable growth.

**Comment by LowKeyNerd** [comment]
If you merge these changes into dev and make the changes I noted before pushing to production, this should be fine. I won't be able to push code changes for a few days myself. I there are any other issues, feel free to update them yourself as I just did this of my own free will / in my own time and don't have much emotional investment :), i.e., you won't hurt my feelings.

File: app/_sections/Architecture.tsx (hunk 1 above)
Comment (fatemeh-ra) [reply app/_sections/Architecture.tsx:49]: Yes its much better
